internal/grpctest/tlogger.go

```diff
@@ -0,0 +1,113 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package grpctest
+
+import (
+	"testing"
+)
+
+var logger = tLogger{infoLog, 2, &testing.T{}}
+
+type logLevel int
+
+// tLogger is a logger used for testing that logs using the testing package's T type.
+type tLogger struct {
+	l logLevel
+	v int
+	t *testing.T
+}
+
+const (
+	infoLog logLevel = iota
+	warningLog
+	errorLog
+	fatalLog
+)
+
+func (g *tLogger) setLevel(l logLevel) {
+	g.l = l
+}
+
+func (g *tLogger) setVerbosity(v int) {
+	g.v = v
+}
+
+func (g *tLogger) logIf(l logLevel, args ...interface{}) {
```

Comment:
We may not need this logic.  I think we'll always want to log everything in our tests.  (Verbosity we might not want to set high by default, however, to avoid spam.)

Reply:
This logic is no longer present.